src/Component/Validator/InArrayValidator.php

```diff
@@ -7,21 +7,49 @@
 
 class InArrayValidator extends AbstractValidator implements OptionsInterface
 {
+    /** @var string */
     public const NAME = 'in_array';
 
     use OptionsTrait;
 
+    /** @var array */
     private $options = [
         'options' => [],
     ];
 
+    /** @var array */
+    private $context = [];
+
+    /** @inheritDoc */
     public function validate($value, array $context = []): void
+    {
+        $this->context = $context;
+
+        if(is_array($value))
+        {
+            $this->validateArray($value);
```

Comment:
dit zou ook moeten werken @vdaruben, beter nog, dit blijft recursief naar strings zoeken.

```php
if (is_array($value)) {
        foreach ($value as $valueItem) {
            return $this->validate($valueItem, $context);
        }
}
```
Ik zag ook een merge confict, check even of je op de laatste versie zit van master. Ik update regelmatig eens maar hopelijk zonder de code te breken.